### PR TITLE
Created german translation for auth.php

### PR DESCRIPTION
**Pull request by sven212**

### CODE REVIEW

File: resources/lang/de/auth.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+return [
+
```
Comment (GrahamCampbell) [inline resources/lang/de/auth.php]: styleci failing due to this whitespace

File: resources/lang/de/auth.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    |
+    | The following language lines are used during authentication for various
+    | messages that we need to display to the user. You are free to modify
+    | these language lines according to your application's requirements.
+    |
+    */
+    
+    'failed'   => 'Diese Zugnangsdaten stimmen nicht mit unseren überien.',
+    'throttle' => 'Zu viele Login-versuche bitte versuche es in :seconds Sekunden erneut.',
+
```
Comment (GrahamCampbell) [inline resources/lang/de/auth.php]: and here

File: resources/lang/de/auth.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    |--------------------------------------------------------------------------
+    | Authentication Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used during authentication for various
+    | messages that we need to display to the user. You are free to modify
+    | these language lines according to your application's requirements.
+    |
+    */
+
```
Comment (GrahamCampbell) [inline resources/lang/de/auth.php]: yeh...

**Comment by GrahamCampbell** [comment]
https://styleci.io/analyses/zYL1Aq

**Comment by jbrooksuk** [comment]
If you re-open this @sven212, we'll fix it up.

**Comment by ApfelUser** [comment]
@jbrooksuk I pushed a passing **PR** at #1069

**Pull request closed without merging**